### PR TITLE
[MXNET-613] Upgrade jetpack to version 3.2.1

### PR DESCRIPTION
**Pull request by KellenSunderland**
## Description ##
Jetson builds are currently failing to install due to some missing dependencies.  We can fix the dependency issue and upgrade Jetpack to 3.2.1.

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [x] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### CODE REVIEW

File: ci/docker/Dockerfile.build.jetson [hunk 1]
```diff
@@ -50,19 +50,19 @@ COPY --from=cudabuilder /usr/local/cuda /usr/local/cuda
...
+    ARM_CUDNN_DEV_INSTALLER_PACKAGE=libcudnn7-dev_7.0.5.15-1+cuda9.0_arm64.deb && \
     wget -nv $JETPACK_DOWNLOAD_PREFIX/$ARM_CUDA_INSTALLER_PACKAGE && \
     wget -nv $JETPACK_DOWNLOAD_PREFIX/$ARM_CUDNN_INSTALLER_PACKAGE && \
     wget -nv $JETPACK_DOWNLOAD_PREFIX/$ARM_CUDNN_DEV_INSTALLER_PACKAGE && \
     dpkg -i $ARM_CUDA_INSTALLER_PACKAGE && \
     apt-key add /var/cuda-repo-9-0-local/7fa2af80.pub && \
     dpkg -i $ARM_CUDNN_INSTALLER_PACKAGE && \
     dpkg -i $ARM_CUDNN_DEV_INSTALLER_PACKAGE && \
-    apt update -y && apt install -y unzip cuda-libraries-dev-9-0 libcudnn7-dev
+    apt update -y || true && apt install -y unzip cuda-libraries-dev-9-0 libcudnn7-dev
```
Comment (lebeg) [inline ci/docker/Dockerfile.build.jetson]: What does `|| true` do in this case?

Comment (larroy) [reply ci/docker/Dockerfile.build.jetson]: not fail the script I assume. We should add a comment

File: ci/docker/Dockerfile.build.jetson [hunk 2]
```diff
@@ -50,19 +50,19 @@ COPY --from=cudabuilder /usr/local/cuda /usr/local/cuda
...
+    ARM_CUDNN_DEV_INSTALLER_PACKAGE=libcudnn7-dev_7.0.5.15-1+cuda9.0_arm64.deb && \
     wget -nv $JETPACK_DOWNLOAD_PREFIX/$ARM_CUDA_INSTALLER_PACKAGE && \
     wget -nv $JETPACK_DOWNLOAD_PREFIX/$ARM_CUDNN_INSTALLER_PACKAGE && \
     wget -nv $JETPACK_DOWNLOAD_PREFIX/$ARM_CUDNN_DEV_INSTALLER_PACKAGE && \
     dpkg -i $ARM_CUDA_INSTALLER_PACKAGE && \
     apt-key add /var/cuda-repo-9-0-local/7fa2af80.pub && \
     dpkg -i $ARM_CUDNN_INSTALLER_PACKAGE && \
     dpkg -i $ARM_CUDNN_DEV_INSTALLER_PACKAGE && \
-    apt update -y && apt install -y unzip cuda-libraries-dev-9-0 libcudnn7-dev
+    apt install -y cuda-libraries-dev-9-0 libcudnn7-dev
```
Comment (larroy) [inline ci/docker/Dockerfile.build.jetson]: we don't need unzip? is installed in another script I guess

**Comment by larroy** [comment]
LGTM

**Comment by marcoabreu** [comment]
Seems like you opened to Box of Pandora :P http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/incubator-mxnet/detail/PR-11494/11/pipeline/796

```
+ apt-get install -y openjdk-8-jre

Reading package lists...

Building dependency tree...

Reading state information...

openjdk-8-jre is already the newest version (8u171-b11-0ubuntu0.16.04.1).

openjdk-8-jre set to manually installed.

0 upgraded, 0 newly installed, 0 to remove and 52 not upgraded.

+ echo 'deb https://dl.bintray.com/sbt/debian /'

+ tee -a /etc/apt/sources.list.d/sbt.list

deb https://dl.bintray.com/sbt/debian /

+ apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2EE0EA64E40A89B84B2DF73499E82A75642AC823

Executing: /tmp/tmp.BQK2RhtT4W/gpg.1.sh --keyserver

hkp://keyserver.ubuntu.com:80

--recv

2EE0EA64E40A89B84B2DF73499E82A75642AC823

gpg: requesting key 642AC823 from hkp server keyserver.ubuntu.com

gpgkeys: key 2EE0EA64E40A89B84B2DF73499E82A75642AC823 can't be retrieved

gpg: no valid OpenPGP data found.

gpg: Total number processed: 0

gpg: keyserver communications error: keyserver helper general error

gpg: keyserver communications error: unknown pubkey algorithm

gpg: keyserver receive failed: unknown pubkey algorithm
```

**Comment by KellenSunderland** [comment]
@marcoabreu I know right!  Tracking this one as well.  We're thinking of work arounds.  Other key servers unfortunately don't have this key.

**Comment by KellenSunderland** [comment]
@marcoabreu This one should be ready if you're ok with the changes.

**Pull request merged**